Cargo.toml

```diff
@@ -1,6 +1,8 @@
 [package]
 name = "numtracker"
-version = "0.1.1"
+# This version is *not* used for the built container image but it should be
+# updated whenever the repository is tagged so that the versions stay in sync.
```

Comment:
https://doc.rust-lang.org/cargo/reference/config.html#command-line-overrides

Can you modify the action that does the build for the release? I do not know if version is a privileged value here.
```cargo --config package.version={{version}}```

Reply:
I think `cargo --config` overrides cargo configuration not project configuration so doesn't help here. I *think* we could use something like [cargo-edit](https://github.com/killercup/cargo-edit#cargo-set-version) to bump the version before building the image and then leave the version here as 0.0.0. As far as I can see there isn't an existing cargo-edit action though.

Reply:
Setting up `cargo-edit` seemed like too much of a faff so it's using `sed` instead and hoping that it's available on ubuntu-latest.

Reply:
sed is available, it's what the helm update was using prior to finding that app-version can be overriden from the command line